src/screens/style.css

```
@@ -0,0 +1,109 @@
+body {
+    background-color: #B1E1B3;
+    font-family: 'varela';
+}
+
+#userImg {
```

camelCase se usa en Javascript (para generar código), para css (generar estilos) se utiliza otro estándar.
```suggestion
.user-img {
```